### PR TITLE
modify template to proper toml format

### PR DESCRIPTION
**Pull request by JinxCappa**
Existing config that is generated is not processed correctly with latest version of frp (v0.58.1) due to formatting.

**IMPORTANT: Please do not create a Pull Request without creating an issue first!**

_(Any change needs to be discussed before proceeding. Failure to do so may result in the rejection of the pull request)._

### Summary
This PR fixes the toml configuration generated in the configmap by the operator

### Type of Change
This PR fixes/implements the following **bugs/features**:

- #5 

### How has this been tested?
Proof: Can be provided if necessary

### Checklist:

- [x] My code follows the code style of this project.

### CODE REVIEW

File: pkg/client/utils/template.go [hunk 1]
```diff
@@ -43,9 +42,9 @@ transport.useEncryption = true
 {{ end }}
 
 {{ if eq $upstream.Type 2 }}
-name = {{ $upstream.Name }}
-type = udp
-localIP = {{ $upstream.TCP.Host }}
+name = "{{ $upstream.Name }}"
+type = "udp"
+localIP = "{{ $upstream.TCP.Host }}"
```
Comment (yejiayu) [inline pkg/client/utils/template.go:47]: $upstream.UDP.Host

File: pkg/client/utils/template.go [hunk 2]
```diff
@@ -43,9 +42,9 @@ transport.useEncryption = true
...
 localPort = {{ $upstream.TCP.Port }}
```
Comment (yejiayu) [inline pkg/client/utils/template.go:48]: $upstream.UDP.Port

File: pkg/client/utils/template.go [hunk 3]
```diff
@@ -43,9 +42,9 @@ transport.useEncryption = true
...
 remotePort = {{ $upstream.TCP.ServerPort }}
```
Comment (yejiayu) [inline pkg/client/utils/template.go:49]: $upstream.UDP.ServerPort

**Review by yejiayu — CHANGES_REQUESTED** [review]

File: pkg/client/utils/template.go (hunk 1 above)
Comment (JinxCappa) [reply pkg/client/utils/template.go]: fixed other fields as well as they also referred to TCP

**Pull request merged**